### PR TITLE
chore(dcos-ui): bump DC/OS UI master+v2.18.0

### PR DESCRIPTION
**Pull request by weblancaster**
## High-level description

What features does this change enable? What bugs does this change fix?

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS-39896](https://jira.mesosphere.com/browse/DCOS-39896) Add Region Information to Services Struct
  - [DCOS-39897](https://jira.mesosphere.com/browse/DCOS-39897) add regions table column
  - [DCOS-40691](https://jira.mesosphere.com/browse/DCOS-40691) in services table, region column uses Tooltip component

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by weblancaster** [comment]
@mesosphere-mergebot label Ready for Review

File: packages/dcos-ui/buildinfo.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "single_source": {
     "kind": "url_extract",
-    "url": "https://downloads.mesosphere.io/dcos-ui/master%2Bdcos-ui-v2.3.0.tar.gz",
-    "sha1": "bdf48d7789a1febfb215e66ab962d16f78d9fc21"
+    "url": "https://downloads.mesosphere.io/dcos-ui/master%2Bdcos-ui-v2.18.0.tar.gz",
+    "sha1": "49cf00c2e28d7b37717d552f951026d4ba4d9d8f"
```
Comment (orlandohohmeier) [inline packages/dcos-ui/buildinfo.json]: ✋ The sha1 checksum is wrong.

**Comment by juliangieseke** [comment]
@mesosphere-mergebot bump-ee

**Comment by juliangieseke** [comment]
Arm flaky Test: https://jira.mesosphere.com/browse/DCOS-21339?jql=text%20~%20%22test_networking.test_vip%22 - do we have to override non-required tests?

File: packages/dcos-ui/buildinfo.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
 {
   "single_source": {
     "kind": "url_extract",
-    "url": "https://downloads.mesosphere.io/dcos-ui/master%2Bdcos-ui-v2.14.0.tar.gz",
-    "sha1": "590a8ec93b5f869f427cd1f650efa6465c3efe1f"
+    "url": "https://downloads.mesosphere.io/dcos-ui/master%2Bdcos-ui-v2.18.0.tar.gz",
+    "sha1": "a299e6bb35365757c5e9c785b08afecc3838f7de"
```
Comment (tnguyen0) [inline packages/dcos-ui/buildinfo.json]: Just curious, where is this sha from? I can't find it anywhere.

**Comment by tnguyen0** [comment]
Got it, thanks @Fabs!

**Pull request merged**